config/sources/families/include/meson64_common.inc

```diff
@@ -39,15 +39,11 @@ GOVERNOR=${GOVERNOR:-ondemand}
 case $BRANCH in
 
 	current)
-		declare -g KERNEL_MAJOR_MINOR="6.1" # Major and minor versions of this kernel. For mainline caching.
-		KERNELBRANCH='branch:linux-6.1.y'
-		KERNELPATCHDIR='meson64-current'
+		declare -g KERNEL_MAJOR_MINOR="6.1"
```

Comment:
Same here.

Reply:
Same explanation as above. There needs to be some separation -- otherwise we'd need to bump _all families_ at the same time, which is not humanly possible for a single person.

Reply:
> otherwise we'd need to bump all families at the same time, which is not humanly possible for a single person.

No, that is not possible. Holding back what doesn't work, might work better. That's all.